kmk/modules/combos.py

```diff
@@ -65,6 +68,15 @@ def matches(self, key):
             return False
 
 
+class Coord(Combo):
+    def coord_matches(self, int_coord: int):
```

Comment:
See previous comment:
```suggestion
class Coord(Combo):
    def matches(self, key: Key, int_coord: int):
```